Whiskey/Whiskey.psd1

```diff
@@ -12,7 +12,7 @@
     RootModule = 'Whiskey.psm1'
 
     # Version number of this module.
-    ModuleVersion = '0.35.0'
+    ModuleVersion = '0.36.0'

```

Comment:
I think the version needs to be 0.37.0 since I inadvertently published a 0.36.0 version prematurely.